src/app/pages/SearchResultsPage/ScopedSearchResultsView.tsx

```diff
@@ -40,7 +35,7 @@ export const ScopedSearchResultsView: FC<{
   const hasMainnetResults = otherResults.some(isOnMainnet)
   const notificationTheme = themes[hasMainnetResults ? Network.mainnet : Network.testnet]
 
-  useRedirectIfSingleResult(wantedScope, wantedResults)
+  useRedirectIfSingleResult(wantedScope, [...wantedResults, ...otherResults])
```

Comment:
I think an easier to understand flow would be:
```tsx
const searchResults = allSearchResults.filter(isOnTheRightParatime) // Completely ignore other paratimes
useRedirectIfSingleResult(searchResults, searchResults)

const wantedResults = searchResults.filter(isInWantedScope)
const otherResults = searchResults.filter(isNotInWantedScope)
```

Reply:
or move `.filter(isOnTheRightParatime)` into parent

Reply:
Moved to parent

Reply:
Did you not mean to change to `useRedirectIfSingleResult(wantedScope, searchResults)`?

Reply:
Right. Cleanup at https://github.com/oasisprotocol/explorer/pull/484 , since this is already merged.